nomenclature/codelist.py

```diff
@@ -134,6 +134,17 @@ def cast_variable_components_args(cls, v, values):
 
         return v
 
+    @validator("mapping")
+    def check_stray_tag(cls, v):
+        """Check that no '{' are left in codes after tag replacement"""
+        for code in v:
+            if "{" in code:
+                raise ValueError(
+                    f"Unexpected {{}} in codelist : {code}."
```

Comment:
Is there a reason for the extra whitespace?
```suggestion
                    f"Unexpected {{}} in codelist: {code}."
```